OpenRA.Game/Network/OrderIO.cs

```diff
@@ -52,6 +55,22 @@ IEnumerable<Order> ParseData(World world)
 			}
 		}
 
+		// We check if the orders are valid (related actors must in world)
+		// before executing, in order to keep consistent with ParseData()
+		// used by network clients.
```

Comment:
```suggestion
		/// <summary>Lightweight version of serialising and deserialising orders</summary>
```

Perhaps such a comment would be more fitting

Reply:
Or actually it would be better to just place it inside `GetOrders`. This function isn't really supposed to be reusable

Reply:
> Or actually it would be better to just place it inside GetOrders. This function isn't really supposed to be reusable

Then we have to make an empty list to store the refreshed string and convert it to array later, I think the `yield return` and `IEnumerable<Order>` can be more pref on that.

Reply:
Also this function is not for `serialising`, it kinda like `deserialising`

Reply:
I think remove the comment here can be better, since we move it to `GetOrders` we don't have to repeat it twice

Reply:
comment removed